src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -158,6 +158,14 @@ internal virtual ImmutableArray<LocalSymbol> GetDeclaredLocalsForScope(CSharpSyn
         }
 
         /// <summary>
+        /// Get local functions declared immediately in scope represented by the node.
+        /// </summary>
+        internal virtual ImmutableArray<LocalFunctionSymbol> GetDeclaredLocalFunctionsForScope(CSharpSyntaxNode node)
```

Comment:
My instinct is to remove it.  But I'd let @VSadov chime in here if this is an existing pattern we deploy in the code base.